content/millennium/r4/other/health-cards.md

```diff
@@ -65,6 +66,35 @@ _Authorization Types_
 
 The OAuth2 token must include `Patient.read` and `Immunization.read` scopes.
 
+### Laboratory Test Result Credentials (DM)
+
+Represents the patient and the clinical data related to a laboratory test result
+
+Contains the following resources:
+
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-dm.html#tab-ms) profile with the following fields, if valued:
+  * [Patient official name](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.name){:target="_blank"}
+  * [Date of Birth](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.birthDate){:target="_blank"}
+
+* One or more Observation resources that follow the [Covid19LaboratoryResultObservationDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-dm.html) profile with the following fields, if valued:
+  * [Status](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.status){:target="_blank"}
+  * [Code](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.code){:target="_blank"}
+  * [Subject](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.subject){:target="_blank"}
+  * [Effective date time](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.effectiveDateTime){:target="_blank"}
+  * [Performer (Organization that performed the action)](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.performer){:target="_blank"}
```

Comment:
@Mohammad-Isyroqi-Fathan I think we need to leave off performer for now. We don't know when that work will be done to return the organization as a performer.

Reply:
Removed in this commit https://github.com/cerner/fhir.cerner.com/pull/687/commits/651682229ecbbbfa9a3aefe97595da0d34e7d464